src/blockdata/script.rs

```diff
@@ -216,6 +216,57 @@ impl Script {
     /// Creates a new empty script
     pub fn new() -> Script { Script(vec![].into_boxed_slice()) }
 
+    /// Generates P2PK-type of scriptPubkey
+    pub fn with_pk(pubkey: &PublicKey) -> Script {
+        Builder::new()
+            .push_key(pubkey)
+            .push_opcode(opcodes::all::OP_CHECKSIG)
+            .into_script()
+    }
+
+    /// Generates P2PKH-type of scriptPubkey
+    pub fn with_pkh(pubkey_hash: &PubkeyHash) -> Script {
```

Comment:
Ditto here, the name should indicate that this is a pay-to-pubkeyhash scriptpubkey